assets/templateFloxEnv/manifest.toml

```diff
@@ -20,13 +20,21 @@
 # pass-in = "$some-env-var" 
 
 # An activation hook will be run when entering the environment.
-# You can define one in 'hook' table with an inline 'script' field,
+# You can define one in the 'hook' table inline via the 'script' field,
 # or provide a file using the 'file' field.
 # If 'file' is provided, the path should be relative to this file.
 # If both 'file' and 'script' are provided,
-# only the hook defined in 'file' will be run 
+# only the hook defined in 'file' will be run .
 
 [hook]
 # script = """
 #   echo "it's gettin flox in here";
 # """
+
+# An environment that works on one system is guaranteed to work on the
+# same type of system,
```

Comment:
nonblocking question/nit: how are you wrapping, looks like sembr at 70? I think just wrapping at 80 looks better for a user facing comment like this

Reply:
For this file specifically I was just going with existing wrapping behavior. I agree that 80 characters looks better, but the lines in the rest of the file are wrapped much shorter than that.